src/assets/scss/application.scss

```diff
@@ -150,11 +150,16 @@ $govuk-new-link-styles: true;
   }
 }
 
+$nav-link-padding: govuk-spacing(3);
```

Comment:
@danacotoran  tagging you so you've visibility of this file changing. Not sure if you are reviewing this PR

Reply:
sorry i think this is just a merge conflict issue